solutions/quickstart/outputs.tf

```diff
@@ -56,3 +56,18 @@ output "master_status" {
   description = "The current status of the Kubernetes master node in the cluster."
   value       = module.ocp_base.master_status
 }
+
+output "next_steps_text" {
+  value       = "Now, you can use Openshift Cluster to build, deploy, and manage containerized applications."
+  description = "Next steps text"
+}
+
+output "next_step_primary_label" {
+  value       = "Go to Openshift Cluster dashboard"
+  description = "Primary label"
+}
+
+output "next_step_primary_url" {
+  value       = "https://cloud.ibm.com/containers/cluster-management/clusters/${module.ocp_base.cluster_id}/overview"
```

Comment:
This is not the Openshift dashbaord. This is the IBM Cloud cluster console page. The Openshift dashboard is based off the ingress domain. Perhaps we could keep this as a secondary one though